ingestion/src/main/java/feast/ingestion/utils/StoreUtil.java

```diff
@@ -246,4 +263,74 @@ public static void checkRedisConnection(RedisConfig redisConfig) {
     }
     jedisPool.close();
   }
+
+  /**
+   * Ensures Cassandra is accessible, else throw a RuntimeException. Creates Cassandra keyspace and
+   * table if it does not already exist
+   *
+   * @param cassandraConfig Please refer to feast.core.Store proto
+   */
+  public static void setupCassandra(CassandraConfig cassandraConfig) {
+    List<InetSocketAddress> contactPoints =
+        Arrays.stream(cassandraConfig.getBootstrapHosts().split(","))
+            .map(host -> new InetSocketAddress(host, cassandraConfig.getPort()))
+            .collect(Collectors.toList());
+    Cluster cluster = Cluster.builder().addContactPointsWithPorts(contactPoints).build();
+    Session session;
+
+    try {
+      String keyspace = cassandraConfig.getKeyspace();
+      KeyspaceMetadata keyspaceMetadata = cluster.getMetadata().getKeyspace(keyspace);
+      if (keyspaceMetadata == null) {
+        log.info("Creating keyspace '{}'", keyspace);
+        Map<String, Object> replicationOptions =
+            cassandraConfig.getReplicationOptionsMap().entrySet().stream()
+                .collect(Collectors.toMap(Entry::getKey, Entry::getValue));
+        KeyspaceOptions createKeyspace =
+            SchemaBuilder.createKeyspace(keyspace)
+                .ifNotExists()
+                .with()
+                .replication(replicationOptions);
+        session = cluster.newSession();
+        session.execute(createKeyspace);
+      }
+
+      session = cluster.connect(keyspace);
+      // Currently no support for creating table from entity mapper:
+      // https://datastax-oss.atlassian.net/browse/JAVA-569
+      Create createTable =
+          SchemaBuilder.createTable(keyspace, cassandraConfig.getTableName())
+              .ifNotExists()
+              .addPartitionKey(CassandraMutation.ENTITIES, DataType.text())
+              .addClusteringColumn(CassandraMutation.FEATURE, DataType.text())
+              .addStaticColumn(CassandraMutation.VALUE, DataType.blob());
```

Comment:
Is the `value` column supposed to be **static**?
https://docs.datastax.com/en/dse/6.0/cql/cql/cql_using/refStaticCol.html

| entities        | feature           | value  |
| ------------- |:-------------:| -----:|
| featureset1:1:entity1=1    | feature1 | SAME VALUE |
| featureset1:1:entity1=1      | feature2      |   SAME VALUE |

I believe this will cause the values for **different** features to be the **same**, which is not what we expect.

Reply:
Good catch, thank you! Fixing it now.

Reply:
Cool thanks. Looks good now